README.md

```diff
@@ -1,5 +1,20 @@
 # dgrants
 
+## Directory Structure
+.
+├── app                     # Frontend Vue.js application
+├── contracts               # Smart contracts
+├── dapp                    # The dapp
```

Comment:
We can remove the `dapp` row since we don't have this folder. The dapp lives in the `app` folder 🙂

Also these blocks should be enclosed by triple backticks in each README since you can see they don't render properly

![image](https://user-images.githubusercontent.com/17163988/126406133-ecd12ead-f0a4-4483-8bb3-f77ffdcfd369.png)

Reply:
Removed.